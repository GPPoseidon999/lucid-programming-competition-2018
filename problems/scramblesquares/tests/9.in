5 5
2 8 6 1
0 9 8 3
0 5 5 7
5 0 8 7
7 0 5 2
7 3 8 5
7 2 6 8
7 7 0 8
8 8 5 1
1 9 4 2
4 8 2 0
8 3 7 4
0 3 0 3
1 7 9 5
0 5 2 9
4 1 8 2
7 8 0 3
3 9 7 0
6 2 3 9
8 5 0 9
6 2 0 5
0 3 1 7
2 2 0 8
0 5 6 2
8 0 4 0
