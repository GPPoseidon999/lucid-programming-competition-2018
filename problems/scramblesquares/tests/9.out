\ 7 /\ 7 /\ 7 /\ 1 /\ 0 /
5 X 33 X 88 X 22 X 99 X 5
/ 8 \/ 0 \/ 6 \/ 4 \/ 2 \
\ 8 /\ 0 /\ 6 /\ 4 /\ 2 /
4 X 33 X 99 X 22 X 11 X 8
/ 7 \/ 8 \/ 3 \/ 8 \/ 6 \
\ 7 /\ 8 /\ 3 /\ 8 /\ 6 /
2 X 00 X 00 X 99 X 55 X 2
/ 5 \/ 4 \/ 7 \/ 0 \/ 0 \
\ 5 /\ 4 /\ 7 /\ 0 /\ 0 /
7 X 00 X 88 X 77 X 33 X 3
/ 8 \/ 2 \/ 0 \/ 1 \/ 0 \
\ 8 /\ 2 /\ 0 /\ 1 /\ 0 /
1 X 88 X 22 X 55 X 77 X 5
/ 5 \/ 0 \/ 6 \/ 9 \/ 5 \
